### ISSUE TITLE
Pre-work Checklist: Developer: Matthew Arofin

### ISSUE THREAD

**Issue by matt-arofin** [hackforla/website#3063]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Action Items
- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your Gmail address with the team lead so you can be added to the Google Drive
- [x] Also, confirm with the team lead that they have added you to the Zoom meeting calendar invites
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a team lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [ ] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
  - [x] Use the information from part 2 to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment.
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue.  This update should be done every week for every issue.  The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points
- Then, add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting
- You may also add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question
- Lastly, you can post your question in the hfla slack channel and link the issue you're working on, so other volunteers can see and respond

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by SAUMILDHANKAR** [comment]
@matt-arofin Just noticed that you haven't checked off the action item corresponding to familiarizing with Figma. Please let us know if you have questions or need help. Will reopen this issue but feel free to close it once that is done. 
Also, noticed that you haven't picked up your next issue and it has been a while now. Please let us know if you are still planning to pick your next good second issue or your availability has changed. Thank you.

**Comment by SAUMILDHANKAR** [comment]
@matt-arofin Planning to close this issue due to inactivity. Please let us know when your availability changes. Will leave a slack message and email as well. Thank you.

**Comment by SAUMILDHANKAR** [comment]
Update: Heard back from Matthew on Slack. Would be updating availability later this week. Also, planning to pick up an issue next.

**Comment by matt-arofin** [comment]
Progress: Reviewed the Figma design system page and familiarised myself with the practices for developers. Will be closing this issue and choosing a new one.
Availability: 3 hours 06/05/22

**Comment by tamara-snyder** [comment]
Hi @matt-arofin, I am reopening your Pre-Work because you still need to do a progress update in the issue you are assigned to and report your time spent so far in this Pre-Work Checklist. Please keep your Pre-Work open until you have done these things. Please use the following format for providing a progress update:
```
1. Progress
2. Blockers
3. Availability
4. ETA
```
See [this section of the HfLA Wiki](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#25-reporting-progress-on-your-issue) for more details.
I have unchecked the items for the progress update and time spent so far. Please check them off again once you've made the appropriate comments.

**Comment by matt-arofin** [comment]
Update on my first issue closed (#2793):

1. Progress - task completed, pull request merged
2. Availability - 2 hours daily, Wednesday, Friday, Saturday
3. Time Spent on pre-work so far - 6.5 hours

**Comment by kathrynsilvaconway** [comment]
@matt-arofin Hi Matt, are you still interested in working with Hack for LA? If so, go ahead and comment your availability to finsih your checklist. If I don't hear back, I will just close this issue. Thanks!

**Comment by matt-arofin** [comment]
Hi @kathrynsilvaconway,I'm still interested, but my availability will be limited. Will block off time to close this issue and pick up a new one next week 😃

Availability:
09/05/22 - 1 hour
09/07/22 - 1 hour
09/08/22 - 2 hours
09/09/22 - 2 hours

**Comment by HackforLABot** [comment]
## Activity Log: matt-arofin
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- matt-arofin opened: https://github.com/hackforla/website/issues/3063 at 04/18/2022, 07:51 PM PDT
- matt-arofin assigned: https://github.com/hackforla/website/issues/3063#issuecomment-1101940486 at 04/18/2022, 07:52 PM PDT
- matt-arofin assigned: https://github.com/hackforla/website/issues/2793#issuecomment-1045507195 at 04/23/2022, 03:37 PM PDT
- matt-arofin opened: https://github.com/hackforla/website/pull/3079 at 04/24/2022, 06:19 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/2793#issuecomment-1107875899 at 04/24/2022, 09:40 AM PDT
- matt-arofin issue closed by PR 3079: https://github.com/hackforla/website/issues/2793#event-6499097985 at 04/26/2022, 06:56 AM PDT
- matt-arofin PR was merged: https://github.com/hackforla/website/pull/3079#event-6499098163 at 04/26/2022, 06:56 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/pull/3086#issuecomment-1114319079 at 05/01/2022, 12:43 PM PDT
- matt-arofin submitted review: https://github.com/hackforla/website/pull/3086#pullrequestreview-958643514 at 05/01/2022, 04:14 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/pull/3085#issuecomment-1114368795 at 05/01/2022, 04:41 PM PDT
- matt-arofin closed as completed: https://github.com/hackforla/website/issues/3063#event-6535445392 at 05/02/2022, 11:34 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/3063#issuecomment-1146643665 at 06/04/2022, 09:17 AM PDT
- matt-arofin assigned: https://github.com/hackforla/website/issues/3215#issuecomment-1145859151 at 06/06/2022, 02:52 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/3215#issuecomment-1147972816 at 06/06/2022, 02:55 PM PDT
- matt-arofin closed as completed: https://github.com/hackforla/website/issues/3063#event-6756420303 at 06/06/2022, 02:56 PM PDT
- matt-arofin opened: https://github.com/hackforla/website/pull/3231 at 06/07/2022, 11:01 AM PDT
- matt-arofin merged: https://github.com/hackforla/website/pull/3231#event-6763645239 at 06/07/2022, 11:06 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/pull/3231#issuecomment-1149013889 at 06/07/2022, 11:17 AM PDT
- matt-arofin reopened: https://github.com/hackforla/website/pull/3231#issuecomment-1149013889 at 06/07/2022, 11:17 AM PDT
- matt-arofin assigned: https://github.com/hackforla/website/issues/3217#issuecomment-1146276918 at 06/08/2022, 01:08 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/3217#issuecomment-1150356512 at 06/08/2022, 01:10 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/pull/3231#issuecomment-1150546791 at 06/08/2022, 05:34 PM PDT
- matt-arofin issue closed by PR 3231: https://github.com/hackforla/website/issues/3215#event-6773929902 at 06/08/2022, 05:42 PM PDT
- matt-arofin PR was merged: https://github.com/hackforla/website/pull/3231#event-6773929907 at 06/08/2022, 05:42 PM PDT
- matt-arofin unassigned: https://github.com/hackforla/website/issues/3217#issuecomment-1153359796 at 06/12/2022, 06:27 PM PDT
- matt-arofin assigned: https://github.com/hackforla/website/issues/2931#issuecomment-1059838534 at 06/16/2022, 12:25 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/2931#issuecomment-1158058333 at 06/16/2022, 12:32 PM PDT
- matt-arofin opened: https://github.com/hackforla/website/pull/3301 at 06/25/2022, 08:15 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/2931#issuecomment-1166308764 at 06/25/2022, 08:18 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/pull/3301#issuecomment-1166364774 at 06/25/2022, 02:32 PM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/3063#issuecomment-1186550699 at 07/17/2022, 08:48 AM PDT
- matt-arofin issue closed by PR 3301: https://github.com/hackforla/website/issues/2931#event-7007744080 at 07/17/2022, 10:19 AM PDT
- matt-arofin PR was merged: https://github.com/hackforla/website/pull/3301#event-7007744085 at 07/17/2022, 10:19 AM PDT
- matt-arofin commented on: https://github.com/hackforla/website/issues/3063#issuecomment-1234696977 at 09/01/2022, 12:32 PM PDT
- matt-arofin issue closed as completed: https://github.com/hackforla/website/issues/3063#event-7363093123 at 09/11/2022, 04:25 PM PDT
